.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       FLAVOR: 'eng'
       PB_OFFICIAL: 'true'
       CHANGELOG: |
-        - Initial Official Build
+        - 2nd Official Build
```

Comment:
Add disabled DM Verity to changelog please

Reply:
ok